src/main/kotlin/no/nav/familie/ba/soknad/api/config/ApplicationConfig.kt

```diff
@@ -41,9 +39,13 @@ internal class ApplicationConfig {
 
     @Bean
     fun apiKeyInjectingClientInterceptor(@Value("\${PDL_API_APIKEY}") pdlApiKey: String,
-                                         @Value("\${PDL_API_URL}") pdlBaseUrl: String): ClientHttpRequestInterceptor {
-        val apiKeyPdlMap = mapOf(Pair(URI.create(pdlBaseUrl), Pair(apiKeyHeader, pdlApiKey)))
-        return ApiKeyInjectingClientInterceptor(apiKeyPdlMap)
+                                         @Value("\${PDL_API_URL}") pdlBaseUrl: String,
+                                         @Value("\${STS_APIKEY}") stsApiKey: String,
+                                         @Value("\${STS_URL}") stsBaseUrl: String): ClientHttpRequestInterceptor {
+        val map = mapOf(
+                Pair(URI.create(pdlBaseUrl), Pair(apiKeyHeader, pdlApiKey)),
+                Pair(URI.create(stsBaseUrl), Pair(apiKeyHeader, stsApiKey)))
```

Comment:
Det er vel forsøkt litt av hvert underveis her. Er denne endringen nødvendig å ha med denne nå som den skal settes i StsRestClient-klassen fra felles? Vil tro at begge gjør samme ting.

Reply:
Denne endringen trengs vel for at api-key skal legges på @oyvindhagen

Reply:
Ah, nå skjønte jeg hva du mente - fjerner den 👍